### PR TITLE
Rename failure metrics

### LINKED ISSUE #58: "distributor_ingester_appends_total" bound three times
* https://github.com/weaveworks/prism/blob/502cbc58f4df12fd6f351635b22ef422f2cae317/distributor.go#L103
* https://github.com/weaveworks/prism/blob/502cbc58f4df12fd6f351635b22ef422f2cae317/distributor.go#L108
* https://github.com/weaveworks/prism/blob/502cbc58f4df12fd6f351635b22ef422f2cae317/distributor.go#L118

### PR DESCRIPTION
**Pull request by jml**
Fixes #58

---
This change is [Reviewable](https://reviewable.io/reviews/weaveworks/prism/65)

### CODE REVIEW

**Comment by juliusv** [comment]
Oh wow. What. I was wondering how this was even possible, given that the metrics registry should complain about two metrics with the same name. Turns out, I just didn't even collect those metrics at all :( Shame on me. Could you also add the missing four ingester metrics to the `Describe()` and `Collect()` methods of the distributor?

**Comment by jml** [comment]
Yeah, was going to do that in a different PR but can do it here also.

File: ring/ring.go [hunk 1]
```diff
@@ -61,17 +61,17 @@ func New(client CoordinationStateClient) *Ring {
...
 		done:   make(chan struct{}),
 		ingesterOwnershipDesc: prometheus.NewDesc(
-			"prometheus_distributor_ingester_ownership_percent",
+			"prism_distributor_ingester_ownership_percent",
 			"The percent ownership of the ring by ingester",
 			[]string{"ingester"}, nil,
 		),
 		ingesterTotalDesc: prometheus.NewDesc(
-			"prometheus_distributor_ingesters_total",
+			"prism_distributor_ingesters_total",
```
Comment (juliusv) [inline ring/ring.go]: Sorry, while you're at it: this shouldn't have a `_total` suffix, as it's a gauge, not a counter. Just `prism_distributor_ingesters`.

File: ring/ring.go [hunk 2]
```diff
@@ -61,17 +61,17 @@ func New(client CoordinationStateClient) *Ring {
...
 			"Number of ingesters in the ring",
 			nil, nil,
 		),
 		tokensTotalDesc: prometheus.NewDesc(
-			"prometheus_distributor_tokens_total",
+			"prism_distributor_tokens_total",
```
Comment (juliusv) [inline ring/ring.go]: Same.

**Comment by juliusv** [comment]
:+1: otherwise.

**Pull request merged**